docs/manage/connections-integrations/webhook-connections/pagerduty.md

```diff
@@ -116,6 +140,24 @@ The URL and supported payload are different based on the version of the PagerDut
  	      }
      }
      ```
+1. Under the **Recovery Payload**:
+   * You can update the `details` section, if you want to customize the PagerDuty recovery notification. Below is the default v1 payload
+   * Note: Do not update the `service_key` , `event_type`, and `incident_key` fields, otherwise recovery notifications will not be generated.
+   ```json
+   {
+	"service_key": "{{ServiceKey}}",
+	"event_type": "resolve",
+	"incident_key": "{{IncidentKey}}",
+	"description": "{{Name}} has recovered at {{TriggerTime}}",
+	"details": {
+		"name": "{{Name}}",
+		"time": "{{TriggerTimeRange}}",
+		"triggerCondition": "{{TriggerCondition}}",
+		"query": "{{Query}}"
+	      }
+    }
+     ```
```

Comment:
```suggestion
   * You can update the `details` section, if you want to customize the PagerDuty recovery notification. Below is the default v1 payload.
   ```json
   {
	"service_key": "{{ServiceKey}}",
	"event_type": "resolve",
	"incident_key": "{{IncidentKey}}",
	"description": "{{Name}} has recovered at {{TriggerTime}}",
	"details": {
		"name": "{{Name}}",
		"time": "{{TriggerTimeRange}}",
		"triggerCondition": "{{TriggerCondition}}",
		"query": "{{Query}}"
	      }
    }
    ```
  :::note
  Do not update the `service_key`, `event_type`, and `incident_key` fields, otherwise recovery notifications will not be generated.
  :::
```